src/defaultOperations.ts

```diff
@@ -0,0 +1,45 @@
+import axios from 'axios';
+class DefaultOperations<T> {
```

Comment:
I rather to another name for this class.
for example: CrudDefaultOperations or just CrudOperations